probe-rs-mi/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "probe-rs-mi"
+version.workspace = true
```

Comment:
I would version this crate separately, bumping probe-rs does not necessarily require bumping the MI types.